src/dispatchers/androidDispatcher.ts

```diff
@@ -30,14 +30,21 @@ export class AndroidDispatcher extends Dispatcher<Android, channels.AndroidIniti
       devices: devices.map(d => new AndroidDeviceDispatcher(this._scope, d))
     };
   }
+
+  async setDefaultTimeoutNoReply(params: channels.AndroidSetDefaultTimeoutNoReplyParams) {
+    this._object.setDefaultTimeout(params.timeout);
+  }
 }
 
 export class AndroidDeviceDispatcher extends Dispatcher<AndroidDevice, channels.AndroidDeviceInitializer> implements channels.AndroidDeviceChannel {
   constructor(scope: DispatcherScope, device: AndroidDevice) {
     super(scope, device, 'AndroidDevice', {
       model: device.model,
-      serial: device.serial
+      serial: device.serial,
+      webViews: device.webViews()
```

Comment:
If you do this, webviews will not be cleaned up together with AndrodiDevice, because they end up in the parent's scope. I'd suggest to emit 'webViewAdded' for all initial webviews.